packages/site/src/components/__tests__/__snapshots__/App.spec.js.snap

```diff
@@ -912,6 +912,77 @@ exports[`App renders correctly 1`] = `
             "slug": "icon",
             "title": "Icon",
           },
+          "link": Object {
+            "behavior": "behavior dummy text for the link component",
+            "design": "design theory about the link component",
+            "doc": Object {
+              "default": Object {},
+            },
+            "examples": Array [
+              Object {
+                "backgroundColor": "#f5f7fa",
+                "description": "an anchor tag",
+                "scope": Object {
+                  "Link": [Function],
+                },
+                "source": "<Link href=\\"https://example.com\\">Make it so</Link>",
+                "title": "Basic link",
+              },
+              Object {
+                "backgroundColor": "#f5f7fa",
+                "description": "A Link will render any children.",
+                "scope": Object {
+                  "Link": [Function],
+                },
+                "source": "<Link href=\\"http://example.com\\">
+          <img src=\\"https://www.fillmurray.com/100/300\\" />
+        </Link>",
+                "title": "Arbitrary children",
+              },
+              Object {
+                "hideSource": true,
+                "scope": Object {
+                  "DemoLayout": [Function],
+                  "Link": [Function],
+                  "simulate": [Function],
+                },
+                "source": "<DemoLayout>
+          <Link href=\\"http://example.com\\">Don't Panic</Link>
+          <Link {...simulate('hover')} href=\\"http://example.com\\">Hover</Link>
+          <Link {...simulate('focus')} href=\\"http://example.com\\">Focus</Link>
+          <Link {...simulate('active')} href=\\"http://example.com\\">Active</Link>
+        </DemoLayout>",
+                "title": "States",
+              },
+            ],
+            "slug": "link",
+            "title": "Link",
+          },
+          "world": Object {
```

Comment:
I think you need to update this snapshot after rebasing from master. `Hello`, `World`, and `HelloWorld` no longer exist.